content/postgresql/concepts/constraints/constraints.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: 'constraints'
+Description: 'Constraints are defined for tables or columns in PostgreSQL. Constraints can be used to control data in order to define limits for inserting, updating and deleting data.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Constraints'
+  - 'Database'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'getting-started-off-platform-for-data-science'
+  - 'paths/design-databases-with-postgresql'
+  - 'paths/computer-science'
+---
+In PostgreSQL, **constraints** are conditions applied to database tables and/or columns. Contraints help to ensure data integrity, consistency and accuracy as well as quality and reliability of the stored data. The constraint rules define limits for insertion, updating and deleting of data. There are different constraint types.
+## Check Constraints
+- A check constraint is a condition that must be fulfilled in all rows of the table.
+```sql
+--Example:
+CREATE TABLE students (
+  student_id SERIAL PRIMARY KEY,
+  age INT,
+  --Is age between 18 and 45?
+  CONSTRAINT age_test CHECK (age BETWEEN 18 AND 45)
+  );
```

Comment:
CREATE TABLE students (
  student_id SERIAL PRIMARY KEY,
  age INT,
  --Is age between 18 and 45?
  CONSTRAINT age_test CHECK (age BETWEEN 18 AND 45)
  );